cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -202,7 +202,29 @@ protected TimeSeries getTimeseries(String page, int pageSize, String names, Stri
 			Field<String> param = DSL.upper(CWMS_UTIL_PACKAGE.call_SPLIT_TEXT(tsId,
 					DSL.val(BigInteger.valueOf(2L)), DSL.val("."),
 					DSL.val(BigInteger.valueOf(6L))));
-			SelectSelectStep<Record8<String, String, String, BigDecimal, String, String, String, Integer>> metadataQuery = dsl.select(
+
+			// What is the syntax for selecting tzName and offsetField from the same subquery?
+			// It works when each field comes from its own subquery.
+			// This didn't work.
+			//			Field<?>[] fields = DSL.select(AV_CWMS_TS_ID2.INTERVAL_UTC_OFFSET.as("INTERVAL_UTC_OFFSET"),
+			//					AV_CWMS_TS_ID2.TIME_ZONE_ID.as("TIME_ZONE_ID"))
+			//					.from(AV_CWMS_TS_ID2).where(AV_CWMS_TS_ID2.CWMS_TS_ID.eq(tsId))
+			//					.fields();
+			//			Field<BigDecimal> offsetField = (Field<BigDecimal>) fields[0];
+			//			Field<String> tzName = (Field<String>) fields[1];
+
+			Field<BigDecimal> offsetField = DSL.select(AV_CWMS_TS_ID2.INTERVAL_UTC_OFFSET.as("INTERVAL_UTC_OFFSET"))
```

Comment:
It seems like these two fields should be fetchable from a single sub-select.  Looking for suggestions of a better way.

Reply:
what's there seems fine to me. Unless there's a very specific performance issue you see as a result I wouldn't worry about it too much.